src/transaction.js

```diff
@@ -47,6 +47,11 @@ function Transaction (network = networks.bitcoin) {
     this.vShieldedOutput = []
     this.bindingSig = 0
   }
+  if (coins.isDash(network)) {
+    // Dash version = 3
+    this.type = 0
+    this.extra_payload = 0
```

Comment:
We use camelCase in this lib so this new field should follow that convention. Will also fix in the future PR mentioned below. Should have caught this in review.

Reply:
The extra payload should be set to `Buffer(0)` for type consistency. At this point I am just taking notes for my next PR :) Review will be more thorough next time. Apologies.

Reply:
See #40